### PR TITLE
Stopgap to head off questions about theming and AWS config

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-3290](https://openedx.atlassian.net/browse/DOC-3290)

Trying to head off some of the questions on theming and potentially AWS configuration
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Subject matter expert: @mattdrayer 
- [x] Subject matter expert: @nedbat 
- [x] Doc team review (sanity check): @edx/doc
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Squash commits

### CODE REVIEW

**Comment by nedbat** [comment]
:+1:

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 1]
```diff
@@ -121,13 +121,18 @@ retrieval, the Open edX platform now uses the Django file storage API. In
...
 
-.. ==================
-.. Updates to Theming
-.. ==================
+==========================
+Themes for Multiple Sites
+==========================
+
+With this release, it is now possible to configure themes for multiple sites
+in a single installation of the Open edX platform.
```
Comment (mattdrayer) [inline en_us/open_edx_release_notes/source/eucalyptus.rst:133]: Do we ever refer the reader to our Slack channels for community support?  If so we could direct them to the #theming channel.

**Comment by catong** [comment]
@lamagnifica LGTM 👍

File: en_us/open_edx_release_notes/source/eucalyptus.rst (hunk 1 above)
Comment (lamagnifica) [reply en_us/open_edx_release_notes/source/eucalyptus.rst:133]: @mattdrayer how is the following addition? (Short answer: no, we send people in the community to the open.edx.com portal, which then sends them to the Google mailing lists and Slack. But if you can handle the increased volume in that Slack channel in the meantime, that's great.)

Comment (mattdrayer) [reply en_us/open_edx_release_notes/source/eucalyptus.rst:133]: I think the addition is fine as a temporary measure -- there is no SLA for questions submitted on the Slack channel, but I've observed that the community has been active in helping people to resolve their issues.

**Comment by lamagnifica** [comment]
@mattdrayer ok to merge?

**Comment by mattdrayer** [comment]
Yes!  👍

**Comment by mattdrayer** [comment]
except there is a failing check?

**Comment by lamagnifica** [comment]
unrelated to this PR, but I am looking into it!

**Pull request merged**